lib/index.d.ts

```diff
@@ -367,6 +367,8 @@ declare namespace Realm.Sync {
     function removeAllListeners(name?: string): void;
     function removeListener(regex: string, name: string, changeCallback: (changeEvent: ChangeEvent) => void): void;
     function setLogLevel(logLevel: 'all' | 'trace' | 'debug' | 'detail' | 'info' | 'warn' | 'error' | 'fatal' | 'off'): void;
+    function setFeatureToken(token: string): void;
+    // FIXME: remove setAccessToken after 2.0.
```

Comment:
Isn't it better to mark this as @deprecated and add an issue for 2.0 to make the relevant change against the 2.0 branch. Fixmes are forgotten by definition

Reply:
I have created https://github.com/realm/realm-js/issues/1300